recipes/labjackpython/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "LabJackPython" %}
+{% set org = "labjack" %}
+{% set upstreamversion = "4-24-2014" %}
+{% set version = "20140424" %}
```

Comment:
Agreed - that would be useful. I looked at ``strftime`` formatting in jinja but didn't see any obvious way to do this, and versions are fairly infrequent, so don't think doing it separately here is the worst thing in the world. Can you think of a better way to get dates into jinja?